master/custom/builders.py

```diff
@@ -279,6 +279,9 @@
     # FreBSD x86-64 clang
     # FreeBSD 15 is CURRENT: development branch (at 2023-10-17)
     ("AMD64 FreeBSD15", "opsec-fbsd15", UnixBuild),
+
+    # riscv64 GCC
+    ("riscv64 Ubuntu23", "onder-riscv64", UnixBuild),
```

Comment:
This list is for Tier-3, but to get Tier-3, you need one core dev responsible of this platform. I suggest to start with No-Tier for now, it's not a big difference.